src/Components/EnergyCalculator/LandingPage/LandingPage.css

```
@@ -0,0 +1,18 @@
+.energyCalculator-container {
+  text-align: center;
+  margin: 2rem 0;
+  width: 100%;
+  padding: 0 2rem;
+}
+
+.energyCalculator-body-text {
+  margin: 2rem 0;
+  text-align: start;
+}
```

Class names should be in Kebab case
```suggestion
.energy-calculator-container {
  text-align: center;
  margin: 2rem 0;
  width: 100%;
  padding: 0 2rem;
}

.energy-calculator-body-text {
  margin: 2rem 0;
  text-align: start;
}
```